src/IK/include/BipedalLocomotion/IK/QPInverseKinematics.h

```diff
@@ -234,8 +238,7 @@ class QPInverseKinematics : public IntegrationBasedIK
      * kinematics. In case of issues, an empty BipedalLocomotion::System::VariablesHandler
      * and an invalid pointer will be returned.
      */
-    static std::pair<BipedalLocomotion::System::VariablesHandler,
-                     std::unique_ptr<QPInverseKinematics>>
+    static IntegrationBasedIKProblem
```

Comment:
If the return type has changed, I guess that also the documentation should be updated.